ocaml/xapi/repository.ml

```diff
@@ -24,6 +24,7 @@ module UpdateIdSet = Set.Make (String)
 
 let capacity_in_parallel = 16
 let reposync_mutex = Mutex.create ()
+let updates_in_cache = ref None
```

Comment:
I would use a lock when accessing / setting this ref - can you use `reposync_mutex` for this purpose perhaps?

Reply:
The `pool.allowed_operation` will protect this cache.

Reply:
Please can you add a comment to mention that?

Reply:
Sure. Thanks.